sal_interfaces/CBP/CBP_Commands.xml

```diff
@@ -24,35 +24,23 @@
         <Subsystem>CBP</Subsystem>
         <Version>3.8</Version>
         <Author>ecoughlin</Author>
-        <EFDB_Topic>CBP_command_moveAzimuth</EFDB_Topic>
-        <Alias>moveAzimuth</Alias>
+        <EFDB_Topic>CBP_command_move</EFDB_Topic>
+        <Alias>move</Alias>
         <Device></Device>
         <Property></Property>
         <Action></Action>
         <Value></Value>
         <Explanation>http://sal.lsst.org</Explanation>
         <item>
             <EFDB_Name>azimuth</EFDB_Name>
-            <Description>The amount to move the azimuth by in absolute.</Description>
+            <Description>The amount to move in azimuth axis.</Description>
```

Comment:
This makes it sound like an offset (yes it's a pre-existing issue but please fix it as part of this PR). Consider something like "The desired azimuth angle".